libbeat/template/load.go

```diff
@@ -156,6 +157,10 @@ func (l *ESLoader) Load(config TemplateConfig, info beat.Info, fields []byte, mi
 // template if it exists. If you wish to not overwrite an existing template
 // then use CheckTemplate prior to calling this method.
 func (l *ESLoader) loadTemplate(templateName string, template map[string]interface{}) error {
+	if sameTemplate, _ := l.sameAsExistingTemplate(templateName, template); sameTemplate {
```

Comment:
We must not ignore the error here, it should be at least logged at the debug level.